apps/taxes/src/lib/graphql.ts

```diff
@@ -43,3 +44,13 @@ export const createClient = (url: string, getAuth: AuthConfig<IAuthState>["getAu
       multipartFetchExchange,
     ],
   });
+
+export const createUrqlClientFromAuthdata = ({
+  saleorApiUrl,
+  token,
+}: {
+  saleorApiUrl: string;
+  token: string;
```

Comment:
good, but even better if you do 

```
{
  saleorApiUrl,
  token,
}: Pick<AuthData, 'saleorApiUrl' | 'token'>
```

so you reference actual type, but only pick what you need

Reply:
done